modules/ai/assets/js/editor/components/prompt-error-message.js

```diff
@@ -1,25 +1,79 @@
-import { Alert } from '@elementor/ui';
+import { Alert, Button } from '@elementor/ui';
 
-const messages = {
-	default: __( 'Unknown error. Please try again later.', 'elementor' ),
-	service_outage_internal: __( 'Service outage. Please try again later.', 'elementor' ),
-	invalid_connect_data: __( 'Invalid connect data.', 'elementor' ),
-	not_connected: __( 'Not connected.', 'elementor' ),
-	quota_reached_trail: __( 'You have reached your quota. Upgrade to unlock more', 'elementor' ),
-	quota_reached_subscription: __( 'You have reached your quota. Upgrade to unlock more', 'elementor' ),
-	rate_limit_network: __( 'Rate limit. Please try again later.', 'elementor' ),
-};
+const PromptErrorMessage = ( { error, onRetry = () => {}, ...props } ) => {
+	const messages = {
+		default: {
+			text: <h3>{ __( 'Unknown error. Please try again later.', 'elementor' ) }</h3>,
```

Comment:
Use the component `<AlertTitle>` instead of `<h3>` + maybe move it to the component and keep here only the text.